_pytest/terminal.py

```diff
@@ -155,13 +160,37 @@ def _tw(self):
                       stacklevel=2)
         return self.writer
 
+    @staticmethod
+    def _new_tw_write(msg, **kw):
```

Comment:
Of course this is hackish. 😉 

I believe the correct thing would be to add this functionality to `py.io.TerminalWriter` instead, what do you guys think?

Reply:
before i tried to kill TerminalWriter altogether - it deserves a general removal as its quite bad as an api and people regularly try to obtain it in a hackish was inside of pytest